### DISCUSSION TITLE
QRemeshify 1.1.0

### DISCUSSION THREAD

**Discussion by ksami** [ksami/QRemeshify#8] [Announcements]
- Performance improvements
- Added guidance of edge flow using material and face set boundaries
- Increase visibility of scale factor (density) option

<hr /><em>This discussion was created from the release <a href='https://github.com/ksami/QRemeshify/releases/tag/'>QRemeshify 1.1.0</a>.</em>

**Comment by Denyalife** [comment]
Guides for material and face set boundaries must be output separately for each option
After generating the new mesh, you need to assign the material and face set to the respective polygons
Enabled Preprocess excludes any kind of guides (sharp, material/face set boundaries) except those that the system itself considers to be sharp corners. But disabling Preprocess may not be desirable in many cases
![Capture](https://github.com/user-attachments/assets/094887e8-d2a3-4fe2-b25f-3d7d277fc106)

**Reply by Denyalife** [reply]
In complex and dense meshes, Preprocess gives worse results. For everyday use, especially in sketching, performance needs to be improved
AMD Ryzen 7 1700 Eight-Core Processor 3.00 GHz
(RAM): 32.0 GB
![untitled3](https://github.com/user-attachments/assets/a8889c53-8f97-41e1-9a87-63709478e906)

**Comment by YouEunSang** [comment]
The remesh addon includes a native library (QuadWild) compiled against GLIBC 2.29 or higher, but my system uses GLIBC 2.28, so it cannot be loaded.

Could you please provide a build compatible with GLIBC 2.28 or lower (e.g. manylinux2014)?

Thank you!
